v1/contracts/UnsignedFixedPoint.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/**
+ * @title Library for fixed point arithmetic on uints
+ */
+library UnsignedFixedPoint {
```

Comment:
Maybe add a TODO or file an issue for operations against non-fixed point numbers (regular uints)?

Also, we currently use signed fixed point numbers on occasion in `TokenizedDerivative`, so we might want to file an issue for adding that. Once we do that, we'd probably also want to implement casting between them as a part of these libraries - maybe a `.signed()` and `.unsigned()` methods on each.

Finally, we'll probably also want an issue related to testing this with echidna and/or manticore.

Reply:
Added TODO for the first bit (I'll work on that as part of my next PR).

Signed: Yep, we'll definitely need that. Do you think that should be part of this library (i.e., make this library FixedPoint.sol that has both `FixedPointUint` and `FixedPointInt`), or as a separate FixedPointInt.sol library?

Totally agree that Echidna and Manticore tests will be useful here. Filed a bug.